### ISSUE TITLE
Loudspeaker icons are confusing

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/talk-android#224]
The current loudspeaker vs internal speaker icons are confusing. Internal speaker uses what I guess is an ear, and the speakerphone uses a "sound system box" icon.

First off they are quite difficult to identify (and I have never seen them before anywhere in Android or in another app), and second they are not symmetric, so you have to basically try it out to find what it does.

Let’s instead use the volume_up and icon: https://material.io/tools/icons/?search=volume&style=baseline, like for example WhatsApp also does:
![](https://cdn.ndtv.com/tech/whatsapp_audio_call_generic.jpg?output-quality=80)

(And we do the same in the iOS Talk app, cc @ivansss)

**Comment by mario** [comment]
What two icons specifically? You mention only volume up which makes little sense to me :(

Btw. the two icons used are material icons.

**Comment by jancborchardt** [comment]
Both Signal and the phone app only use the volume_up icon. When it's active, it's marked with an outline:
![](https://i.imgur.com/YQiqWWS.jpg)

So since we use a button it should probably be "pressed down".

**Comment by mario** [comment]
Well, it's not really a button. How about we use volume down/volume up @jancborchardt ?

**Comment by skjnldsv** [comment]
I agree, this is confusing :)

**Comment by mario** [comment]
@skjnldsv no, I wanted your thoughts on the solution - pressed button is a bit meh. If you both insist on volume_up, I can do that - but I'd prefer volume_up and volume_down then.

**Comment by mario** [comment]
![screenshot_20180703-103335_nextcloud talk](https://user-images.githubusercontent.com/5810/42208514-a4eefe94-7eac-11e8-8e09-0572b86e9c05.jpg)

@skjnldsv

**Comment by skjnldsv** [comment]
I agree with the volume up as well.
I won't really mind design wise, but whatsapp use the same as android stock, messenger... which is a button toggle with a darker background but the icon is the same no matter what state :)
So I'm on the side of the UX unification across all apps if we can ;)

**Comment by Ivansss** [comment]
On iOS we use the following icons.

Loud speaker disabled:
![screen shot 2018-07-03 at 13 16 34](https://user-images.githubusercontent.com/4638605/42216862-90d52188-7ec3-11e8-9b43-2ae8347311d7.png)

Loud speaker enabled:
![screen shot 2018-07-03 at 13 16 45](https://user-images.githubusercontent.com/4638605/42216883-9ee7048a-7ec3-11e8-8fec-898fd6dc10ea.png)

**Comment by mario** [comment]
But @Ivansss - this also makes little sense to me as to me that means NO SOUND.
